### DISCUSSION TITLE
ECU-C, wifi and reboot

### DISCUSSION THREAD

**Discussion by lopezmathieu** [ksheumaker/homeassistant-apsystems_ecur#189] [Q&A] [answered]
Hello, 
In the prerequisites, i understand that the reboot is ok on ECU-C only with wired connexion. 
https://github.com/ksheumaker/homeassistant-apsystems_ecur#prerequisites
Unfortunatly, i have a wireless connexion. Is the reboot ok wireless ? 
Thanks, Mathieu

**Comment by HAEdwin** [comment, marked as answer]
Yes, as far as the reboot is concerned, it does not depend on a wired or unwired connection

**Reply by lopezmathieu** [reply]
Hi, thanks, i will try it, have a good day
